docs/apache-airflow-providers-microsoft-azure/operators/azure_synapse.rst

```diff
@@ -41,6 +41,17 @@ Below is an example of using this operator to execute a Spark application on Azu
       :end-before: [END howto_operator_azure_synapse]
 
 
+AzureSynapseRunPipelineOperator
+-----------------------------------
+Use the :class:`~airflow.providers.microsoft.azure.operators.synapse.AzureSynapseRunPipelineOperator` to execute a pipeline application within Synapse Analytics.
+By default, the operator will periodically check on the status of the executed pipeline to terminate with a "Succeeded" status.
```

Comment:
This is not the right description for this operator. Probably a copy-paste mistake as you used `AzureSynapseRunSparkBatchOperator` for template.
```suggestion
The operator will Executes a Synapse Pipeline.
```